src/config/inversion-fixes.config

```diff
@@ -2240,6 +2240,13 @@ NO INVERT
 
 ================================
 
+tn.gov
```

Comment:
It seems like it's at the wrong position:

```
   -   "tn.gov",
        "toggl.com",
        "tpondemand.com",
        "tproger.ru",
        "translate.yandex.ru",
        "travis-ci.org",
        "trip101.com",
    +   "tn.gov",
    ```
    
    It should be moved above tn.gov.